README.md

```diff
@@ -92,6 +92,34 @@ Signature types:
         env: prod
   ``` 
 
+5. Certificate. It will verify that the image has been signed using the Certificate provided by the user.
+  The `certificate` must be PEM encoded. Optionally the settings can have
+  the list of PEM encoded certificates that can create the `certificate_chain`
+  used to verify the given `certificate`.
+  The `require_rekor_bundle` should be set to `true` to have a stronger
+  verification process. When set to `true`, the signature must have a Rekor
+  bundle and the signature must have been created during the validity
+  time frame of the `certificate`.
+
+  ```yaml
+  signatures:
+    - image: "registry-testing.svc.lan/kubewarden/pod-privileged:v0.1.9"
+      certificate: |
+        -----BEGIN CERTIFICATE-----
+        XXX
+        -----END CERTIFICATE-----
+      certificate_chain:
+      - |
+        -----BEGIN CERTIFICATE-----
+        XXX
```

Comment:
```suggestion
        <intermediate cert>
```

Maybe worth pointing here or somewhere that the chain should have intermediates first, root last.